lib/docker.js

```diff
@@ -595,7 +600,7 @@ function main(options, configuration) {
         // set the defaults
         targets = {
             debian: 'debian:jessie',
-            ubuntu: 'ubuntu:14.04'
+            ubuntu: 'ubuntu:18.04'
```

Comment:
Let's not do changes that don't belong conceptually in this PR. Also, the last Ubuntu version we supported in prod was 14.04 so better to keep it there (even though we don't use Ubuntu in production any more at all).

Reply:
Reverted back